subworkflows/local/quantify_rsem/main.nf

```diff
@@ -19,32 +21,54 @@ workflow QUANTIFY_RSEM {
     //
     // Quantify reads with RSEM
     //
-    RSEM_CALCULATEEXPRESSION ( reads, index )
-    ch_versions = ch_versions.mix(RSEM_CALCULATEEXPRESSION.out.versions.first())
+    if (use_sentieon){
 
+        SENTIEON_RSEMCALCULATEEXPRESSION ( reads, index )
```

Comment:
For this and similar code changes in this PR, since usage is identical could we reduce maintenance burden like:

```
    if (use_sentieon){
        RSEM_CALCULATE_EXPRESSION = SENTIEON_RSEMCALCULATEEXPRESSION
    }
```

... and avoid duplicating everything?

Reply:
I've seen this work somewhere else

Reply:
I didn't know that works. I'll test it

Reply:
Any chance you recall where you saw this? I tested both, your suggestion and:

```
    if (use_sentieon_star){

        RSEM = SENTIEON_RSEMCALCULATEEXPRESSION
    } else {
        RSEM = RSEM_CALCULATEEXPRESSION
    }
```
    
   Yours just uses the original process then, so sentieon is not called. the second makes nextflow sad: `Missing process or function RSEM()`

Reply:
went on a slack dive to find some other options:

- Conditional imports are a no-no from Ben: https://nextflow.slack.com/archives/C02TH6EH554/p1731783820513199?thread_ts=1731694516.174269&cid=C02TH6EH554
-

Reply:
this may work 🤞 

```
if (use_sentieon_star){

        SENTIEON_RSEMCALCULATEEXPRESSION ( reads, index )
        ch_rsem_out = SENTIEON_RSEMCALCULATEEXPRESSION

    } else {
        RSEM_CALCULATEEXPRESSION ( reads, index )
        ch_rsem_out = RSEM_CALCULATEEXPRESSION
    }
```